resources/views/emails/collaboratoremail.blade.php

```diff
@@ -1,11 +1,49 @@
-Hello!
-<br>
-<br>
-TLCMap user {{$senderemail}} has sent you an invite to collaborate on their dataset as a {{$dsrole}}.
-<br>
-<br>
-Please use the following link to join their dataset: <a href="{{$sharelink}}">{{$sharelink}}</a>
-<br>
-<br>
-<br>
-This is an automated message from <a href="tlcmap.org">TLCMap</a>
\ No newline at end of file
+<table>
+    <tbody>
+        <tr>
+            <td style="box-sizing:border-box;padding:25px 0;text-align:center">
+                <a href=<?php echo config('app.url') ?> style="box-sizing:border-box;color:#bbbfc3;font-size:19px;font-weight:bold;text-decoration:none" target="_blank">

```

Comment:
Please use the blade variable instead of embedded PHP statment.

Reply:
Updated